hstrat/juxtaposition/_impl_column/_calc_rank_of_first_retained_disparity_between_generic.py

```diff
@@ -136,9 +136,10 @@ def advance_second():
         # second has strata ranks beyond the newest found in first
         # conservatively assume mismatch will be with next rank
         assert first_iter is None
-        assert first_prev_rank + 1 == min(
-            first.GetNumStrataDeposited(), second.GetNumStrataDeposited()
-        )
+        assert first_prev_rank is not None
+        # assert first_prev_rank + 1 == min(
```

Comment:
I'd delete entirely if it's understood why this assert is no longer appropriate or add a comment explaining